packages/web/modals/validator-squad.tsx

```diff
@@ -0,0 +1,214 @@
+import {
+  createColumnHelper,
+  flexRender,
+  getCoreRowModel,
+  getSortedRowModel,
+  useReactTable,
+} from "@tanstack/react-table";
+import classNames from "classnames";
+import { observer } from "mobx-react-lite";
+import { FunctionComponent } from "react";
+import { useMemo } from "react";
+
+import { Icon } from "~/components/assets";
+import { CloseButton } from "~/components/buttons";
+import { SearchBox } from "~/components/input";
+import { IS_FRONTIER } from "~/config/index";
+import { ModalBaseProps } from "~/modals/base";
+
+export const ValidatorSquadModal: FunctionComponent<ModalBaseProps> = observer(
+  (props) => <ValidatorSquadContent {...props} />
+);
+
+const data = [
+  ["Cosmostation", "0.00", "8.44%", "5%"],
+  ["Figment", "0.00", "8.44%", "5%"],
+  ["Stargaze", "0.00", "8.44%", "5%"],
+  ["Frens", "0.00", "8.44%", "5%"],
+  ["Figment", "0.00", "8.44%", "5%"],
+  ["interchain.fm", "0.00", "8.44%", "5%"],
+  ["imperator.co", "0.00", "8.44%", "5%"],
+  ["Chorus One", "0.00", "8.44%", "5%"],
+  ["Electric", "0.00", "8.44%", "5%"],
+  ["wosmongton", "0.00", "8.44%", "5%"],
+];
+
+type Validator = {
+  validator: string;
+  myStake: number;
+  votingPower: number;
+  comissions: number;
+};
+
+interface ValidatorSquadContentProps {
+  onRequestClose: () => void;
+  isOpen: boolean;
+}
+
+const ValidatorSquadContent = observer(
+  ({ onRequestClose, isOpen }: ValidatorSquadContentProps) => {
+    const columnHelper = createColumnHelper<Validator>();
+
+    const virtualRows = data;
+    const rows = data;
+
+    const columns = useMemo(
+      () => [
+        columnHelper.accessor((row) => row, {
+          cell: "hello world",
+          header: "Validator",
+          id: "validator",
+          sortDescFirst: false,
+        }),
+        columnHelper.accessor((row) => row, {
+          cell: "hello world",
+          header: "My stake",
+          id: "myStake",
+          sortDescFirst: false,
+        }),
+        columnHelper.accessor((row) => row, {
+          cell: "hello world",
+          header: "Voting power",
+          id: "votingPower",
+          sortDescFirst: false,
+        }),
+        columnHelper.accessor((row) => row, {
+          cell: "hello world",
+          header: "Commissions",
+          id: "comissions",
+          sortDescFirst: false,
+        }),
+      ],
+      [columnHelper]
+    );
+
+    const table = useReactTable({
+      data,
+      // @ts-ignore
+      columns,
+      getCoreRowModel: getCoreRowModel(),
+      getSortedRowModel: getSortedRowModel(),
+      manualSorting: true,
+    });
+
+    const paddingTop = 0;
+    const topOffset = 324;
+
+    const handleSearchInput = () => console.log("search");
+
+    return isOpen ? (
+      <div className="absolute z-50 max-h-[938px] w-full max-w-[1168px] rounded-[20px] bg-osmoverse-800 px-[62px] pt-8 pb-[54px]">
+        <CloseButton
+          className="relative top-0 right-0"
+          onClick={() => onRequestClose()}
+        />
```

Comment:
Note by looking at other modals in that folder, they use a `<ModalBase />` component. It handles the close button and title for you. It can be customized if needed though. I recommend looking at that.